include/seqan3/io/sequence/sequence_file_format.hpp

```diff
@@ -66,10 +73,17 @@ concept bool sequence_file_format_concept = requires (t v)
     };
 
 };
+//!\endcond
 
 namespace detail
 {
 
+/*!
+ * \brief Check whether all given files meet the sequence_file_format_concept.
+ * \tparam index Current file index.
+ * \tparam variant_type Type of data structure.
+ * \return False if at least one file does not meet the sequence file format concept, true otherwise.
```

Comment:
Referring to the manual `\return` `\returns` and `\result` are equivalent. Currently, I find the first two in our code. I change it to `\returns` now and discuss in the next meeting if we want a consistency over all files.